Makefile

```diff
@@ -315,7 +315,7 @@ else
 	go build -mod=readonly $(BUILD_FLAGS) -o build/contract_tests ./cmd/contract_tests
 endif
 
-contract-tests: build build-contract-tests-hooks
+contract-tests: build-linux build-contract-tests-hooks
```

Comment:
why build-linux instead of just build?

Reply:
I tried on my mac :P and it failed because the dockers where it runs need the Linux version of it. I've been working on my ubuntu machine before that's why I haven't noticed it until now

Reply:
but what if I want to run it on my own machine?

Reply:
Should work now

Reply:
it failed because the dockers where it runs need the Linux version of it.

I don't understand. `build` target should build Linux version when run inside Linux

Reply:
The containers are unable to build, I guess they were created as minimal as possible, just able to run a pre-made binary.

The past days I was working on a Linux machine, so this was all good.

Today I'm on my Mac and by purging the build folder before every run, the `make testnet-start` command failed, because I was building our `tendermint` binary for Mac and trying to use it on the Linux containers